hw3/card.js

```diff
@@ -0,0 +1,34 @@
+function Card(name, newCardBalance){          
+    this.name = name;
+    if(isNaN(newCardBalance)){
+        newCardBalance = 0;
+    };    
+    this.name?true:this.name="Anonim Card";       
+    // chekcBalance function    
+    this.getBalance = function (){        
+        return newCardBalance;
+    };       
+    //balance replenishment function
+    this.addBalance = function (summ){
+        newCardBalance += summ;
+        return newCardBalance;
+    };
+    //write-off balance function
+    this.offBalance = function(summ){
+        if(summ > newCardBalance){return "User has not money to write-off"}       
+        else{
+            newCardBalance -= summ;
+            return newCardBalance;
+        }        
+    };
+    //transfer to another currency function
+    this.transferBalance = function(course){
+        return newCardBalance * course;
+    };    
+};
+
+let NikolaenkoCard = new Card(process.env.USERNAME,20);
+module.exports = {
```

Comment:
you export 2 objects. 1- used in tests only and the second one - used only in your logic. The tests are not check anything

Reply:
fixed